tests/functional/conftest.py

```diff
@@ -434,11 +422,8 @@ def user(gl):
 
     yield user
 
-    try:
-        # Use `hard_delete=True` or a 'Ghost User' may be created.
-        user.delete(hard_delete=True)
-    except gitlab.exceptions.GitlabDeleteError as e:
-        print(f"User already deleted: {e}")
+    # Use `hard_delete=True` or a 'Ghost User' may be created.
+    helpers.safe_delete(object=user, hard_delete=True)
```

Comment:
I'm still not sure whether
`helpers.safe_delete(object=user, hard_delete=True)`
is clearer than
`helpers.safe_delete(user, hard_delete=True)`

I guess a good example of my thinking is Python's own `list.sort()` vs `sorted()` that do almost the same thing:
if you're passing an obvious object to it then that can be positional IMO, whereas options are enforced as keyword-only.
See their docs on how that's explicitly enforced here:
https://docs.python.org/3/library/functions.html#sorted
https://docs.python.org/3/library/stdtypes.html#list.sort

Reply:
Haha I guess that's my real question there if that's really always desired.